tools/make/kube.mk

```diff
@@ -128,6 +128,12 @@ prepare-e2e: prepare-helm-repo install-fluent-bit install-loki install-tempo ins
 	kubectl rollout status statefulset tempo -n monitoring --timeout 5m
 	kubectl rollout status deployment otel-collector -n monitoring --timeout 5m
 
+.PHONY: uninstall-e2e
+uninstall-e2e:
```

Comment:
should it be `prepare-e2e-telemetry` and `uninstall-e2e-telemetry` ?

Reply:
done